### PR TITLE
added Authorization token documentation

### PR DESCRIPTION
**Pull request by alfmags**

### CODE REVIEW

File: docs/swagger.json [hunk 1]
```diff
@@ -14,6 +14,54 @@
     }
   ],
   "paths": {
+    "/authorization/get": {
```
[the remark this answers is not in the document]
Comment (cloveras) [reply docs/swagger.json]: The `GET` should be from the method itself: `GET:/authorization`(or similar). 

Btw: Check Stripe: https://stripe.com/docs/api#authentication

```A sample test API key is included in all the examples here, so you can test any example right away. To test requests using your account, replace the sample API key with your actual API key.```

File: docs/swagger.yaml [hunk 2]
```diff
@@ -931,6 +974,25 @@ definitions:
       userDetails:
         description: Vipps User Information
         "$ref": "#/definitions/UserDetails"
+  AuthorizationTokenResponse:
```
Comment (reza1001) [inline docs/swagger.yaml]: response parameters have underscore in their titles. Actual result from a call returned following parameters:
token_type
expires_in
ext_expires_in
expires_on
not_before
resource
access_token

File: docs/swagger.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
     "version": "1.0",
     "title": "Vipps eCommerce APIs"
   },
```
Comment (reza1001) [inline docs/swagger.json]: replace 
"host": "localhost:8080",
with
 "host": "apitest.vipps.no",

Comment (reza1001) [reply docs/swagger.json]: also remove
"basePath": "/v2/",
as the access token url path differs

File: docs/swagger.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (reza1001) [reply docs/swagger.json]: replace:
```
   "paths": {
 +    "/authorization/get": {
```
with:
```
   "paths": {
 +    "/accessToken/get": {
```

File: docs/swagger.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
...nt account is created |\n\n## Authorization Token\n The Authorization token (also called access token) API endpoint provides the JWT Bearer token that needs to be passed in every API request in the authorization header. Merchant application use the <ClientId> and <ClientSecret> to get a JWT access token. JWT access token is a base 64 encoded string value that must be aquire first before making any Vipps api calls. The following headers need to be included in the Authorization Service request.\n\n\n  \n  | **Authorization Service Url** |\n  | :-------------------- \n   | https://api.vipps.no/accessToken/get |\n\n  \n  | **Header Name** | **Header Value** | **Description** |\n  | :-------------------------- | :--------------------------- | :--------------------------------------------- |\n  | client_id | A GUID value| Client ID received when merchant registered the application |\n | Client_secret | Base 64 String | Client Secret received when merchant registered the application \n | Ocp-ApimSubscription-Key | Base 64 encoded string | Subscription key for eCommerce product.<br>This can be found in User Profile page on Merchant developer portal after merchant account is created |\n\n",
```
[the remark this answers is not in the document]
Comment (alfmags) [reply docs/swagger.json]: Fixed and commited,

File: docs/swagger.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (alfmags) [reply docs/swagger.json]: Fixed and commited,

File: docs/swagger.yaml (hunk 2 above)
Comment (alfmags) [reply docs/swagger.yaml]: Fixed and commited,

**Pull request closed without merging**